oregon-a/basket-dev/web-deploy.yaml

```diff
@@ -48,366 +37,226 @@ spec:
         - -c
         env:
         - name: BUILDPACK_URL
-          valueFrom:
-            secretKeyRef:
-              key: buildpack-url
-              name: basket-dev-v107-env
+          value: https://github.com/pmclanahan/heroku-buildpack-python-peep
         - name: CELERY_ALWAYS_EAGER
-          valueFrom:
-            secretKeyRef:
-              key: celery-always-eager
-              name: basket-dev-v107-env
+          value: "False"
         - name: CELERY_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-dev-v107-env
+          value: 4
         - name: CELERY_POOL
-          valueFrom:
-            secretKeyRef:
-              key: celery-pool
-              name: basket-dev-v107-env
+          value: prefork
         - name: DATABASE_URL
           valueFrom:
             secretKeyRef:
               key: database-url
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: DEBUG
-          valueFrom:
-            secretKeyRef:
-              key: debug
-              name: basket-dev-v107-env
-        - name: DEIS_APP
-          valueFrom:
-            secretKeyRef:
-              key: deis-app
-              name: basket-dev-v107-env
-        - name: DEIS_DOMAIN
```

Comment:
This and the DEIS_APP deletion above are blocked by https://github.com/mozmeao/basket/issues/206

Reply:
In which order do you think we should make these changes? Or should we try to coordinate them as close together as possible?

Reply:
On further review, I don't believe the middleware update is a blocker for removing the DEIS_* values. As far as I can tell the missing env vars shouldn't break anything in the [current code](https://github.com/mozmeao/basket/blob/2a653b4e3a67e55b7628289dc86e2bc7cdacbda5/basket/news/middleware.py#L33-L43), the only impact should be that `X-Backend-Server` will only have the HOSTNAME value.